src/layer/AbstractSentinelHubV1OrV2WithCCLayer.ts

```diff
@@ -34,4 +34,10 @@ export class AbstractSentinelHubV1OrV2WithCCLayer extends AbstractSentinelHubV1O
       cloudCoverPercent: tile.cloudCoverPercentage,
     };
   }
+
+  protected getFindDatesAdditionalParameters(): Record<string, any> {
+    return {
+      maxcc: this.maxCloudCoverPercent / 100,
```

Comment:
I couldn't find a reference that it should be divided by 100, but in my tests there was no difference in results for numbers over `1`, so this code seems to be correct. Feel free to resolve this comment unless you wish to investigate further.

Reply:
I will leave that for now. We can quickly fix it, if it will present an issue.